src/main/resources/reference.conf

```diff
@@ -480,6 +480,7 @@ resourceTypes = {
         roleActions = ["notify_admins"]
       }
     }
+    allowLeaving = true
```

Comment:
does this mean that all resources will allow leaving by default? Adn I suppose we can set it differently for different envs?

Reply:
This just enables it for the `managed-group` type (the default value for allowLeaving if not specified for a type is false). We decided to implement this at the resource type level because we didn't want to introduce the functionality to other resource types that might not be expecting this behavior to be possible. We started with workspaces, now groups, and our third and final target will probably be billing projects.

And yeah- we could totally do this on a per-env basis as well if we wanted to have some more granularity